x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -43,11 +43,12 @@ import (
 )
 
 const (
-	requestRetrySleepEnv     = "KIBANA_REQUEST_RETRY_SLEEP"
-	maxRequestRetriesEnv     = "KIBANA_REQUEST_RETRY_COUNT"
-	defaultRequestRetrySleep = "1s"                             // sleep 1 sec between retries for HTTP requests
-	defaultMaxRequestRetries = "30"                             // maximum number of retries for HTTP requests
-	defaultStateDirectory    = "/usr/share/elastic-agent/state" // directory that will hold the state data
+	requestRetrySleepEnv          = "KIBANA_REQUEST_RETRY_SLEEP"
+	maxRequestRetriesEnv          = "KIBANA_REQUEST_RETRY_COUNT"
+	defaultRequestRetrySleep      = "1s"                             // sleep 1 sec between retries for HTTP requests
+	defaultMaxRequestRetries      = "30"                             // maximum number of retries for HTTP requests
+	defaultStateDirectory         = "/usr/share/elastic-agent/state" // directory that will hold the state data
+	defaultFleetPackagePolicyName = "fleet-server-policy"
```

Comment:
Shouldn't we rely on a config parameter instead of hardcoding it?

Reply:
I think configuring this value would be redundant with configuring a policy to report to specifically.

Reply:
I agree with @jlind23 it would just be more flexible and very easy to make it configurable so lets add it. I agree in most cases it should not be changed, but if they want to change it they can.